src/pages/Browse/index.jsx

```diff
@@ -63,7 +74,14 @@ export default function Browse() {
       <div className="browse-list">
         {!loading
           ? gameList?.map((game) => {
-              return <GameCard key={game.id} gameData={game} />;
+              return (
+                <GameCard
+                  key={game.id}
+                  gameData={game}
```

Comment:
can't `Browse` and `MostPopular` can be a same component ? both have almost same code.